src/L.Geoserver.js

```diff
@@ -82,8 +82,9 @@ L.Geoserver = L.FeatureGroup.extend({
         if (typeof that.options.style === "function") {
           for (i = 0; i < layers.length; i++) {
             that.addLayer(layers[i]);
-            if (i.setStyle) {
-              i.setStyle(that.options.style(i.feature));
+            if (layers[i].setStyle) {
```

Comment:
Is this working correctly?

Reply:
so far it works on my site. I use this kind of script.
```
style: function (f) {
        switch (f.properties.color) {
            case 'blue':
                console.log('blue case');
                return {
                    color: "#057af6"
                };
            case 'orange':
                console.log('orange case');
                return {
                    color: "#ec7800"
                };
            case 'red':
                console.log('red case');
                return {
                    color: "#f31869"
                };
            case 'yellow':
                console.log('yellow case');
                return {
                    color: "#f2dd08"
                };
            case 'green':
                console.log('green case');
                return {
                    color: "#00db3e"
                };
            default:
                console.log('default case');
                return {
                    color: "#717071"
                };
        }
    },
```